vault/resource_pki_secret_backend_config_ca.go

```diff
@@ -25,45 +26,64 @@ func pkiSecretBackendConfigCAResource() *schema.Resource {
 			},
 			"pem_bundle": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
```

Comment:
So if we are using KMS then the `pem_bundle` is not required? Not sure why this is now optional.

Reply:
I had initially gotten this from the RFC:

```
we will also make pem_bundle optional, to allow updating the managed key name parameter after a key has already been established, either via the API or directly in the HSM.  This allows for operators to change the configuration in the future.
```

Looking more into the Vault docs however, I realized that this was no longer a necessary change. Reverted in latest commits